README.md

```diff
@@ -372,10 +372,13 @@ Resolves to:
     {
       "enable": true,
       "port": 9856,
-      "httpsPort": 9857
+      "httpsPort": 9857,
+      "excludeRoutes": []
```

Comment:
The HTML minifier has a similar param called `exceptionRoutes`, either this new one, or the old one should be renamed for consistency.